alure.pxd

```diff
@@ -456,8 +485,18 @@ cdef extern from '<AL/alure2.h>' namespace 'alure' nogil:
 
 
     cdef cppclass SourceGroup:
-        pass
-
```

Comment:
Please keep double newlines around classes.  Not exactly part of PEP 8 but the classes here are too long anyway.